0000-getting-started-cmake.md

```diff
@@ -24,11 +24,11 @@ In this getting started guide we will setup your machine to perform local CMake
 
 > ##### Prerequisites for Hermetic Builds
 >
-> For local containerized and hermetic build you need to **install docker** on your system, if you can't install it you can rely on `--remote` and a tipi.build account to run the container on our cloud. Or resort to cached but non-hermetic builds with `--host`.
+> For local containerized and hermetic build you need to **install docker** on your system. Ff you can't install it, you can use the tipi.build cloud instead with `--remote` and a tipi.build account. Finally you can launch non-hermetic builds with `--host` builds but still benefit from local caching.
```

Comment:
Here too we should remove the mention of --host then, we are certainly offering too much possibility in the same page. I think hermetic and remote are very similar so that's fine but --host should be findable on the documentation somewhere else.